schemas/event.yml

```diff
@@ -316,6 +361,17 @@
             that indicate that something was deleted.
             A common example is `event.category:file AND event.type:deletion`
             to indicate that a file has been deleted.
+        - name: denied
+          description: >
+            The denied event type is used for the subset of events within a category that
+            indicate that something was denied. Common examples include
+            `event.category:”network” AND event.type:denied` (to indicate a network
```

Comment:
Please make sure to always use straight quotes, not the fancy ones Google Docs and the others use.

```suggestion
            `event.category:"network" AND event.type:denied` (to indicate a network
```